src/components/NewMessage/NewMessage.vue

```diff
@@ -333,6 +334,7 @@ export default {
 			clipboardTimeStamp: null,
 			typingInterval: null,
 			wasTypingWithinInterval: false,
+			debouncedUpdateChatInput: debounce(this.updateChatInput, 50)
```

Comment:
Not sure methods were defined at this moment, but seems to work. Maybe increase it to 200ms?